environments.json

```diff
@@ -3377,7 +3377,7 @@
   "node13_2": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 13.2+",
+    "short": "Node &gt;=13.2 &lt;14",
     "platformtype": "engine",
     "note_id": "harmony-flag",
     "equals": "chrome79",
```

Comment:
Should we also obsolete Node.js 13 in this PR?

Reply:
no, it won't be EOL until June 2020, see https://github.com/nodejs/Release